packages/react-native-reanimated/android/src/main/cpp/reanimated/android/NativeProxy.h

```diff
@@ -169,8 +169,9 @@ class NativeProxy : public jni::HybridClass<NativeProxy>,
   friend HybridBase;
   jni::global_ref<NativeProxy::javaobject> javaPart_;
   jsi::Runtime *rnRuntime_;
-  std::shared_ptr<ReanimatedModuleProxy> reanimatedModuleProxy_;
   jni::global_ref<LayoutAnimations::javaobject> layoutAnimations_;
+  std::shared_ptr<jni::global_ref<LayoutAnimations::javaobject>> layoutAnimationsShared_;
```

Comment:
Let's hold just the shared_ptr.
```suggestion
// We keep a shared_ptr to a jni::global_ref because you cannot make weak_ptrs from jni::global_ref.
  std::shared_ptr<jni::global_ref<LayoutAnimations::javaobject>> layoutAnimations_;
```